### ISSUE TITLE
about cal the regression loss

### ISSUE THREAD

**Issue by CharmsYANG** [c0nn3r/RetinaNet#19]
hello sir, thx very much for the clear and clean repository. 
there is a small question about cal the regression loss, what's the meaning of those sentences when cal the regression loss in the end of file losses.py?

                targets = targets/torch.Tensor([[0.1, 0.1, 0.2, 0.2]]).cuda()

and 

                regression_loss = torch.where(
                    torch.le(regression_diff, 1.0 / 9.0),
                    0.5 * 9.0 * torch.pow(regression_diff, 2),
                    regression_diff - 0.5 / 9.0
                )

why not cal the L2 loss directily? is there any trick? 

thx again for your attention and reply
